indico/modules/rb_new/client/js/modules/calendar/reducers.js

```diff
@@ -76,17 +126,44 @@ export default combineReducers({
                 return {...state, rows: camelizeKeys(action.data)};
             case calendarActions.ROOM_IDS_RECEIVED:
                 return {...state, roomIds: action.data.slice()};
+            case bookingActions.BOOKING_DELETE_SUCCESS: {
+                const {bookingId, roomId} = camelizeKeys(action.data);
+                const {rows} = state;
+                return {...state, rows: filterDeletedBooking(rows, bookingId, roomId)};
+            }
+            case bookingActions.BOOKING_STATE_UPDATED: {
+                const {booking: {id, roomId}, bookingState} = camelizeKeys(action.data);
+                const {rows} = state;
+                let newRows;
+
+                if (['rejected', 'cancelled'].indexOf(bookingState) !== -1) {
```

Comment:
do you really consider that more readable than a simple `||`?

    if (bookingState === 'rejected' || bookingState === 'cancelled')

BTW, ES6 has `[...].includes(somevalue)`

Reply:
Yeah, I know about `includes` I just remember I had some problems with it at some point.. however I don't remember what it was exactly.